test/ClangImporter/generic_compatibility_alias.swift

```diff
@@ -11,12 +11,12 @@ import ObjCIRExtras
 
 func foo(_: SwiftConstrGenericNameAlias<String>) {
 // expected-error@-1 {{'SwiftConstrGenericNameAlias' requires that 'String' inherit from 'NSNumber'}}
-// expected-note@-2  {{requirement specified as 'T' : 'NSNumber' [with T = String]}}
+// FIXME-note@-2 {{requirement specified as 'T' : 'NSNumber' [with T = String]}}
```

Comment:
There's a few tests like this - should I just switch over to using FileCheck? May be worth adding some more tests to explicitly test this case as well.